1build

```diff
@@ -59,17 +61,25 @@ def run(build_file_name, arguments):
             print(__help_message__(project))
         else:
             command = project.get_command(command_name)
-            execute(command)
+            execute(command, project.before, project.after)
     except ValueError as error:
         print(error)
 
 
-def execute(command):
+def execute(command, before=None, after=None):
+    cmd = command.cmd
+
     print(dash)
     print("Name: " + command.name)
     print("Command: " + command.cmd)
+    if before is not None:
+        print("Before: " + before)
+        cmd = before + "&&" + cmd
```

Comment:
Maybe this should be write something like: ```before + " && " + cmd``` ?

Reply:
Make Sense!